catalog/opensearch/catalog-opensearch-source/src/main/java/org/codice/ddf/opensearch/source/OpenSearchSource.java

```diff
@@ -383,8 +383,9 @@ private SourceResponse doOpenSearchQuery(
       WebClient restWebClient)
       throws UnsupportedQueryException {
     // All queries must have at least a search phrase to be valid
-    searchPhraseMap.putIfAbsent(OpenSearchConstants.SEARCH_TERMS, "*");
-
+    if (searchPhraseMap.isEmpty()) {
```

Comment:
We really only want to do this if everything is empty... no searchPhrase, no temporalSearch, no spatialSearch.
We should be able to represent this:
![image](https://user-images.githubusercontent.com/4419422/40688866-c2a3f7f0-636d-11e8-8dea-3e0ec12e2f60.png)
like this: services/catalog/query?lat=20&lon=20&radius=500
I believe this should be legal

Reply:
I believe so. As long as `parameters` is not empty, the query should be executed just fine. However, with the new fix, it fails some of the opensearch source tests in `OpenSearchSpec.groovy`. Some of these tests require that `q=*` is a parameter for the query. I can alter the tests to reflect the change we want to make, but it's up to you @millerw8